src/components/cryptoList/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  Container,
+  NameContainer,
+  NameCryp,
+  NameSym,
+  LogoContainer,
+  Logo,
+  PriceContainer,
+  Price,
+  Percentage,
+  PreArrow,
+  ArrowD,
+  Separator,
+} from './style';
+import ArrowUp from '../../assets/arrowup.png';
+import ArrowDown from '../../assets/arrowdown.png';
+
+const Cryptoitems = ({item}: any) => {
+  const {Name, symbol, img, market_data} = item;
```

Comment:
You can use the destructuring in the declaration of the function: 

( {item: {Name, symbol, img, market_data}} )